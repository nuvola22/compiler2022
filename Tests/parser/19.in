begin
    writeln(1, 'hello, world', 2.3);
end.