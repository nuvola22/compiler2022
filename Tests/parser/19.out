      program
         begin
            call
               writeln
               1
               'hello, world'
               2,3
